samples/sensor/sht3xd/nrf51_ble400.overlay

```diff
@@ -16,3 +16,9 @@
 		alert-gpios = <&gpio0 2 GPIO_INT_ACTIVE_HIGH>;
 	};
 };
+
+/ {
+	aliases {
+		sht3xd-0 = &sht3xd;
```

Comment:
IIUC, aim of this alias is to instantiate a generic set of defines that could be used to implement  samples/sensor/sht3xd, right?

Reply:
Yes.